shenyu-admin/src/main/java/org/apache/shenyu/admin/discovery/DefaultDiscoveryProcessor.java

```diff
@@ -176,6 +176,7 @@ public void fetchAll(final String discoveryHandlerId) {
         String discoveryId = discoveryHandlerDO.getDiscoveryId();
         if (discoveryServiceCache.containsKey(discoveryId)) {
             ShenyuDiscoveryService shenyuDiscoveryService = discoveryServiceCache.get(discoveryId);
+            // TODO 如果列表为空？
```

Comment:
remove this

Reply:
Thank you for your review, I will modify it immediately.